plugins/vendor/services/ZoomVendorService.php

```diff
@@ -0,0 +1,431 @@
+<?php
+/**
+ * @service zoomVendor
+ * @package plugins.vendor
+ * @subpackage api.services
+ */
+class ZoomVendorService extends KalturaBaseService

```

Comment:
Shouldn't this service be the entry point for other vendor integrations in the future as well?
If implementing it this way we will need to create a new service class for every vendor integration we will do in the future.

Reply:
After talking about it -> agreed to keep like I suggested